mne/preprocessing/ica.py

```diff
@@ -214,7 +214,7 @@ def __init__(self, n_components=None, max_pca_components=None,
         self.max_pca_components = max_pca_components
         self.n_pca_components = n_pca_components
         self.ch_names = None
-        self.random_state = random_state if random_state is not None else 42
+        self.random_state = random_state
```

Comment:
I agree about seeding, but the default used to be fixed to `42` (in some places) and now it defaults to `None`